src/styles/blocks/consultation.scss

```
@@ -0,0 +1,66 @@
+.consultation {
+  padding-top: 150px;
+
+  &__title {
+    @extend %section-title;
+    margin-bottom: 90px;
+  }
+
+  &__block {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__element {
+    flex-basis: 40.4%;
```

40.4? use round numbers
```suggestion
    flex-basis: 40%;
```